docs/export-as-code.mdx

```diff
@@ -4,33 +4,54 @@ title: Export as code
 
-![Stately Studio with the export from code icon highlighted in the toolbox and the code panel open.](/export_from_code_panel.jpg)
+You can currently export your machine as:
 
-1. Open the <Code size={18} /> **Code** panel from the right tool menu.
-2. Use the JSON, JavaScript, or TypeScript buttons to display the code in your preferred format in the text area.
-3. Use the **XState version 5 beta** toggle to toggle between code supported by XState version 4 and XState version 5 beta.
+- JSON code for use with XState
+- JavaScript code for use with XState
+- TypeScript code for use with XState
+- [Mermaid](https://mermaid.js.org) code and diagrams for use in GitHub and anywhere Mermaid is supported
+
+:::new
+
+You can now export to Mermaid code and diagrams.
+
+:::
+
+## How to export as code
+
+You can export as code by:
+- Using the <Code size={18} /> **Code** panel in the right tool menu
+- Using the <MoreHorizontal size={18} /> menu alongside the machine name in the machines list
+- Using the <Upload size={18} /> export icon button in the header
+
+:::studio
+
+You can also [import from code](import-from-code.mdx) in any of these locations.
+
+:::
+
+### Export to JSON, JavaScript, or TypeScript
+
+1. Use one of the options above to option the **Export to code** modal or <Code size={18} /> **Code** panel.
+2. Use the dropdown menu to choose from JSON, JavaScript, or TypeScript.
+3. Use the **XState version 5 beta** toggle to choose between code supported by XState version 4 and XState version 5 beta.
 4. Use the **Copy** button to copy the code to your clipboard.
 5. Paste into your code editor.
 
-## Export code using the header export icon
-
-![Stately Studio with the export icon highlighted in the top bar and the export code modal open.](/export_from_top_bar.jpg)
+### Export to Mermaid
 
-1. Open the export **Code** modal using the <Upload size={18} /> export icon in the header.
-2. Use the JSON, JavaScript, or TypeScript button to copy the code in your preferred format to your clipboard.
-3. Use the **XState version 5 beta** toggle to toggle between code supported by XState version 4 and XState version 5 beta.
-4. Paste into your code editor.
+1. Use one of the options above to option the **Export to code** modal or <Code size={18} /> **Code** panel.
+2. Use the dropdown menu to choose Mermaid.
+3. Use the **Copy** button to copy the code to your clipboard.
+4. Use the **Preview** tab to preview the diagram and **Copy SVG** to copy the diagram SVG to your clipboard.
+5. Paste into your code or graphics editor.
 
-## Export code from the machine menu
+:::tip
 
-![Stately Studio with the export code item highlighted in the machine list’s context menu.](/export_from_machine_list.jpg)
+Wrap your Mermaid code in a fenced code block with the `mermaid` language identifier to [share your diagram on GitHub](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams).
```

Comment:
Suggestion: might be worth linking to GitLab too? https://handbook.gitlab.com/handbook/tools-and-tips/mermaid/

Reply:
Thanks, great suggestion! Done!